analyzers/tests/SonarAnalyzer.Test/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.cs

```diff
@@ -525,6 +525,36 @@ public void Branching_ConditionEvaluated()
         SETestContext.CreateCS(code, check).Validator.ValidateTagOrder("Begin", "End");
     }
 
+    [TestMethod]
+    public void Branching_ConditionEvaluated_IsInLoop()
+    {
+        var code = """
```

Comment:
Nitpick. Same below.
```suggestion
        const string code = """
```

Reply:
EDU: Is there a significant performance benefit to this or why are we doing this?